Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfBufferView.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
+{
+    /// <summary>
+    /// A view into a buffer generally representing a subset of the buffer.
+    /// </summary>
+    [Serializable]
+    public class GltfBufferView : GltfChildOfRootProperty
+    {
+        /// <summary>
+        /// The index of the buffer.
+        /// </summary>
+        public int buffer = -1;
+
+        /// <summary>
+        /// The offset into the buffer in bytes.
+        /// <minimum>0</minimum>
+        /// </summary>
+        public int byteOffset = -1;
+
+        /// <summary>
+        /// The length of the bufferView in bytes.
+        /// <minimum>0</minimum>
+        /// </summary>
+        public int byteLength = -1;
+
+        /// <summary>
+        /// The stride, in bytes, between vertex attributes or other interleavable data.
+        /// When this is zero, data is tightly packed.
+        /// <minimum>0</minimum>
+        /// <maximum>255</maximum>
+        /// </summary>
+        public int byteStride = -1;
+
+        /// <summary>
+        /// The target that the WebGL buffer should be bound to.
+        /// All valid values correspond to WebGL enums.
+        /// When this is not provided, the bufferView contains animation or skin data.
+        /// </summary>
+        public GltfBufferViewTarget target = GltfBufferViewTarget.None;
+
+        public GltfBuffer Buffer { get; internal set; }
```

Comment:
There are a lot of public properties with internal setters like this Buffer throughout the schema namespace. Do we want to provide comment summaries for them since they are public?

Reply:
as much as I prefer doc comments, if this is khronos code, adding the summaries may make it harder to integrate updates, thoug

Reply:
I don't have a strong preference on this one, but would want to aid developers if the worth outweighs the maintenance costs. @StephenHodgson what are your thoughts?

Reply:
No Khronos code, this is a buffer to ease access to the data.

If anything the fields represent the data structure of the glTF standard.

Reply:
Should we add a summary since these items have public getters?

Reply:
The existing summary tagged properties appear to match what's in the Khronos repo, while this undocumented part is new: https://github.com/KhronosGroup/UnityGLTF/blob/master/GLTFSerialization/GLTFSerialization/Schema/BufferView.cs#L44

Reply:
Likely should be the description found in the schema
https://github.com/KhronosGroup/glTF/tree/master/specification/2.0/schema